Makefile

```diff
@@ -72,8 +72,8 @@ nextest-archive-clean:
 # INTEGRATION TESTS
 # ##############################################################################
 
-integration-env-up: emily-cdk-synth
-	docker compose --file docker/docker-compose.test.yml up -d
+integration-env-up: emily-cdk-synth clean-nakamoto-headers
+	docker compose -f docker/docker-compose.test.yml --profile default --profile bitcoin-mempool --profile sbtc-signer up -d
```

Comment:
For integration tests I don't think we want `bitcoin-mempool` (it spawns also mempool api & co), nor the `sbtc-signer`s. We also probably don't want the bitcoin miner,  tx-broadcaster nor some other container in there. Also applies to the CI variant.

If the sbtc-signers are up beacuse of the event observer, it's probably better to handle it mounting different configurations (it should be fine wrt the state, assuming a clean event-observer db).